functionnal_tests/cypress/integration/app_file/file_name_spec.js

```diff
@@ -0,0 +1,49 @@
+import { PAGES } from '../../support/urls_commands'
+
+const threadTitle = 'Thread'
+const fileTitle = 'File'
+const fullFilename = 'Linux-Free-PNG.png'
+const contentType = 'image/png'
+let threadId
+let fileId
+let workspaceId
+
+describe("An app's name", () => {
```

Comment:
I don't understand this label, shouldn't it be "a content's name" ?

Reply:
Maybe "The app File title"?

Reply:
Yes I was thinking "a content's name inside an app", I'll make it clearer.